### PR TITLE
Add kube-linter check

### PR DESCRIPTION
**Pull request by mmorhun**
This PR adds `kube-linter` check that runs only if change to the operator deployment configuration is made.

### CODE REVIEW

File: deploy/operator/base/deployment.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: hacbs-jvm-operator
-          image: hacbs-jvm-operator
+          image: hacbs-jvm-operator:latest
```
Comment (rnc) [inline deploy/operator/base/deployment.yaml:25]: I'm not sure that would help as https://docs.kubelinter.io/#/generated/checks?id=latest-tag would still complain? Would it not be better to exclude this check in the configuration file?

Comment (mmorhun) [reply deploy/operator/base/deployment.yaml]: Changed tag name as @stuartwdouglas suggested.

File: deploy/operator/base/deployment.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: hacbs-jvm-operator
-          image: hacbs-jvm-operator
+          image: hacbs-jvm-operator:next
```
Comment (stuartwdouglas) [inline deploy/operator/base/deployment.yaml:25]: I think the CI failure is legit, and is likely caused by this update https://github.com/redhat-appstudio/jvm-build-service/blob/91a173d2a2421b0d9e169b9678044a9558001aae/deploy/openshift-ci.sh#L22

**Pull request merged**